templates/components/form/networkname.html.twig

```diff
@@ -0,0 +1,166 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2022 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+
+{% import 'components/form/fields_macros.html.twig' as fields %}
+
+{% set no_header = no_header|default(not item.isNewItem() and not _get._in_modal|default(false)) %}
+{% set bg = '' %}
+{% if item.isDeleted() %}
+   {% set bg = 'asset-deleted' %}
+{% endif %}
+
+<div class="asset {{ bg }}">
+   {{ include('components/form/header.html.twig', {'in_twig': true}) }}
+
+   {% set params  = params ?? [] %}
+   {% set no_header = true %}
+   {% set field_options = {
+         'locked_fields': item.getLockedFields(),
+   } %}
+
+   <div class="card-body d-flex flex-wrap">
+      <div class="col-12 col-xxl-12 flex-column">
+         <div class="d-flex flex-row flex-wrap flex-xl-nowrap">
+            <div class="row flex-row align-items-start flex-grow-1">
+               <div class="row flex-row">
+                  {% set field_options = field_options|merge(params) %}
+                  {% set session_get_plural = call('Session::getPluralNumber', []) %}
+                  {% set rand = random() %}
+
+                  {% if ID %}
+                     <input type="hidden" name="itemtype" value="{{ item.fields['itemtype'] }}">
+                     <input type="hidden" name="items_id" value="{{ item.fields['items_id'] }}">
+                  {% endif %}
+
+                  {{ fields.textField(
+                     'name',
+                     item,
```

Comment:
```suggestion
                     item.name,
```

Reply:
Seems not correct. With that change, on NetworkName with an empty name, it's set to `N/A`; which is incorrect, and prevents form to be stored because it's not a valid name.

Reply:
```suggestion
                     item.fields['name'],
```